README.md

```diff
@@ -14,10 +14,11 @@ The library uses an iframe to create a clean environment in which to extract met
 
 ## API
 
-### getNativeMethod(path)
-Fetches a native window method at a given path. A path is simply the string representation of a method's location on the window: eg. window.setInterval.
+### getNativeMethod(path{,  bindContext})
+Fetches a native window method at a given path. A path is simply the string representation of a method's location on the window: eg. window.setInterval. A bindContext is the HTML where you would like to bind the method.
```

Comment:
Could you change the doc to this perhaps? "`bindContext` is an optional object to bind at the context - if not provided, the window/document will be used instead."